extensions/3DTILES_tile_metadata/README.md

```diff
@@ -0,0 +1,113 @@
+# 3DTILES_tile_metadata
+
+## Overview
+
+This extension to 3D Tiles allows for storage of per-tile metadata in external binary buffers.
+
+## Indexing
+
+When using 3D Tiles 1.0, the tile metadata is indexed using the Depth First Traversal of the `tileset.json`.
+
+When using the `3DTILES_implicit_tiling` extension, the tile metadata is indexed using the metadata bitstream.
+
+## Dependencies
+
+This extension depends on [3DTILES_binary_buffers](https://github.com/CesiumGS/3d-tiles/blob/3DTILES_binary_buffers/extensions/3DTILES_binary_buffers/README.md) for storage of the binary data.
+
+## Properties Reference
+
+---------------------------------------
+### 3DTILES_tile_metadata Tileset JSON extension
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**properties**|`object`|An object containing one or more metadata properties associated with each tile.|No|
+|**bufferViews**|`array`|An array containing typed views into buffers|No|
+|**buffers**|`array`|An array of buffers.|No|
+
+---------------------------------------
+### 3DTILES_tile_metadata.properties
+
+This property of the 3DTILES_tile_metadata object enumerates the different metadata properties associated with each tile in the tileset.
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**semantic**|`string`|The name of the property. Semantic names must be unique.|No|
+|**bufferView**|`integer`|The index of the bufferView in the the bufferViews array of `3DTILES_binary_buffers`.|☑️ Yes|
+
+Application-specific semantics may also be defined, with the caveat that they must begin with an underscore, e.g. _CLASSIFICATION.
+
+## Example
+
+```json
+{
+  "extensions": {
```

Comment:
The example should be updated to use `3DTILES_binary_buffers`

Reply:
Updated examples to use 3DTILES_binary_buffers